web/src/components/dashboard/SidebarCardFundingNudge.test.tsx

```diff
@@ -46,8 +78,76 @@ describe("<SidebarCardFundingNudge />", () => {
     card = generateSidebarCardContent({ id: fundingNudge });
   });
 
+  const generateTaxFilingUserData = (params: {
+    publicFiling: boolean;
+    formedInNavigator?: boolean;
+    businessName?: string;
+    responsibleOwnerName?: string;
+    taxId?: string;
+  }): UserData => {
+    let formationData: FormationData = {
+      formationFormData: createEmptyFormationFormData(),
+      formationResponse: undefined,
+      getFilingResponse: undefined,
+      completedFilingPayment: false,
+      businessNameAvailability: undefined,
+      dbaBusinessNameAvailability: undefined,
+      lastVisitedPageIndex: 0,
+    };
```

Comment:
pebble - we have a generateEmptyFormationData function, that I think would be better here. 

That being said, couldn't you just make this let `formationData: FormationData | undefined = undefined` or does that lead to some weird type BS later?

Reply:
[boulder] we should lean into the generators wherever possible. If the structure changes later it will cause this test to fail.